frontend/src/pages/_document.tsx

```diff
@@ -0,0 +1,76 @@
+import RawDocument, {
```

Comment:
This file takes care of rendering SSR Styled Component stylesheet and favicon related things